### ISSUE TITLE
Can't boot after installation

### ISSUE THREAD

**Issue by antonis179** [lopaka/instructions#17]
Hi!

Awesome guide! Very complete and thank you very much for posting it!

I can't seem to get the partitioning right in order to get it to boot though... Could you let me know how you partitioned it?

I did:
EFI: 512MB
EXT4 @ /: ~26GB
SWAP: 4GB
Tried grub on both partitions - no joy.
Tried the auto partitioning -> couldn't install grub.

The install completed fine but no boot device shows up afterwards...
What monumentally stupid thing am I doing wrong?

**Comment by lopaka** [comment]
Hello @antonis179 

Sorry I haven't gotten to this yet.  Where you able to find out the issue?

Thanks!

**Comment by antonis179** [comment]
Hey,

Yeah grub was failing to install but the error message was not displayed if you booted in live mode.
But HAD to boot live in order to erase emmc after every try (otherwise installation got frozen).
This happens on both msdos and gpt tables.

Eventually found this out cause I saw the EFI partition was completely empty.
The system however was getting installed properly so.... I remembered the commands to install manually through a live cd where ~1 page long and went the lazy route and used boot-repair to fix it ^^

This netbook is so damn quirky.
Anyways awesome guide - it's much easier to have everything consolidated in one place.
I'll figure out the sound today (hopefully)
